augur/curate/__init__.py

```diff
@@ -42,6 +43,13 @@ def create_shared_parser():
         """)
     shared_inputs.add_argument("--metadata",
         help="Input metadata file, as CSV or TSV.")
+    shared_inputs.add_argument("--id-column",
+        help="Name of the metadata column that contains the record identifier for reporting duplicate records. "
+             "Uses the first column of the metadata file if not provided.")
+    shared_inputs.add_argument("--duplicate-reporting",
+        choices=[ method.value for method in DataErrorMethod ],
+        default=DataErrorMethod.ERROR.value,
+        help="How should duplicate records be reported.")
```

Comment:
I tried this before but it errors because of mismatch between the choice strings and the type. 

I went down a rabbit hole of argparse choices + enums ([see example stackoverflow](https://stackoverflow.com/questions/43968006/support-for-enum-arguments-in-argparse)), but thought the suggested custom enum methods were very convoluted. I thought just changing `args.duplicate_reporting` to the enum type outside of argparse was easier to understand.